src/services/baseService.ts

```diff
@@ -46,15 +46,21 @@ export abstract class BaseService {
   public getRegion(): string {
     return this.options.region || this.serverless.service.provider.region;
   }
-  
+
   public getStage(): string {
     return this.options.stage || this.serverless.service.provider.stage;
   }
 
+  public getPrefix(): string {
+    return this.config.provider.prefix;
+  }
+
   public getResourceGroupName(): string {
-    return this.options["resourceGroup"]
+    const name = this.options.resourceGroup
       || this.serverless.service.provider["resourceGroup"]
```

Comment:
Consider changing to `this.config.provider.resourceGroup` since the `config` property is now scoped to the `ServerlessAzureConfig` interface.